### PR TITLE
Migration documentation fixes

### PR DESCRIPTION
**Pull request by esarbe**
Add migration documentation for optional JsValues

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @esarbe,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

**Comment by esarbe** [comment]
Is there anything I should to to get it merged?

File: documentation/manual/releases/migration24/Migration24.md [hunk 1]
```diff
@@ -378,6 +378,21 @@ As a result of these changes, your code can now assume that all values of type `
...
+```
+need to be replaced with
+
+```scala
+(JsPath \ "property").readNullable[String]
+```
+
+The same goes for `Writes` with `JsPath.writeNullable` and `Format` with `JsPath.formatNullable`.
+
+Please note that `writeNullable` will return a `JsError` if a node could not be found before the last node. Previously a `JsSuccess(None)` was returned.
```
Comment (studiodev) [inline documentation/manual/releases/migration24/Migration24.md]: Do you mean `readNullable` instead of `writeNullable`?

**Comment by dotta** [comment]
@esarbe Thanks for your contribution. To be able to merge this, please squash your commits and answer @studiodev question.

**Comment by esarbe** [comment]
@studiodev Yes, thanks for catching that. is now fixed.
@dotta updated and squashed.

**Comment by dotta** [comment]
LGTM

**Comment by dotta** [comment]
and backported to 2.4.x https://github.com/playframework/playframework/commit/d488926d83341a8ae255c607457cf785446b4ce2

**Pull request merged**